base/applications/cmdutils/replace/replace.c

```diff
@@ -189,13 +180,12 @@ INT replace(TCHAR source[MAX_PATH], TCHAR dest[MAX_PATH], DWORD dwFlags, BOOL *d
         WriteFile (hFileDest, buffer, dwRead, &dwWritten, NULL);
 
         /* Done! or ctrl break! */
-        if (dwWritten != dwRead || CheckCtrlBreak(BREAK_INPUT))
```

Comment:
Why no CheckCtrlBreak?

Reply:
This is outside of `cmd.exe` and `Ctrl+C` needs to be caught in a control handler.

Reply:
Is it correctly handled now?

Reply:
Yes, I think it is.

Reply:
OK yes I saw after that that you added the ctrl-c handler where you set the boolean to TRUE. This should do the job :+1: